bundle/manifests/konveyor-operator.clusterserviceversion.yaml

```diff
@@ -5,7 +5,86 @@ metadata:
     alm-examples: |-
       [
         {
-          "apiVersion": "tackle.konveyor.io/v1alpha2",
+          "apiVersion": "tackle.konveyor.io/v1alpha1",
+          "kind": "Addon",
+          "metadata": {
+            "name": "myaddon",
+            "namespace": "konveyor-tackle"
+          },
+          "spec": {
+            "container": {
+              "image": "quay.io/konveyor/tackle2-addon-analyzer:latest",
+              "imagePullPolicy": "Always",
+              "name": "myaddon",
+              "resources": {
+                "limits": {
+                  "cpu": 1,
+                  "memory": "1Gi"
+                },
+                "requests": {
+                  "cpu": 1,
+                  "memory": "512Mi"
+                }
+              }
+            },
+            "resources": {
```

Comment:
I don't think we want this `resources` on line 30.

Reply:
Thank you, updated!